misc/protobuf-ballerina/src/main/java/org/ballerinalang/protobuf/cmd/GrpcCmd.java

```diff
@@ -218,6 +219,9 @@ private File createTempDirectory() {
         createTempDirectory(protobufHome);
         File compilerHome = new File(protobufHome, TEMP_COMPILER_DIRECTORY);
         createTempDirectory(compilerHome);
+        File apiHome = new File(googleHome, TEMP_API_DIRECTORY);
+        createTempDirectory(protobufHome);
```

Comment:
```suggestion

```

Reply:
In line no: 219, we are creating the protobuf directory

Reply:
Removed the line